### ISSUE TITLE
Any plan to include non seasonal TV anime?

### ISSUE THREAD

**Issue by c933103** [soruly/trace.moe#21]
Examples of anime content that appears to be not indexed into the project's database now:
- Anime promotional materials that are not part of full seasonal anime:
Examples: Madoka Magica Concept Movie, Prism Nana Pilot PV, Kemono Friends JRA/Nissin collaboration PV
- Short/Flash/Doujin/Unofficial/Official online(niconico/youtube)/blu-ray special/other release channel animation content
Examples: Touhou Musou Kakyou, "Haruhi-chan", "Index-tan", Flash anime version of "Nyaruko", "Kosys", Kataribe Shoujo Honoma
- Music Video in form of and other form of anime produced not independently
Examples: Music Video of Love Live and 22/7 and such, Yuuki Yuuna Pachislot special footage, Kemono Friends Animelo Summer Live 2017 Special Movie
- Non-Japanese anime
Some animation products from Korea or China are also commonly described as anime despite the accuracy of such description would be up for debate.

**Comment by soruly** [comment]
- Anime promotional materials that are not part of full seasonal anime:
Since most PV usually appears with the anime name somewhere, so that was not included in database, but I do have plans to include PV from youtube
- Short/Flash/Doujin/Unofficial/Official online(niconico/youtube)/blu-ray special/other release channel animation content
I think these should also be included, but since the source varies and is hard to collect and organize, it takes time to include all. Currently my priority is TV>Bluray/DVD>SPs>ONAs>PV>others but priority is usually given to popularity.
- Music Video in form of and other form of anime produced not independently
I'd prefer to exclude Events / Live from database, since it isn't really useful for most people. But Anime Music MV like "Shelter" would be indexed
- Non-Japanese anime
basically it depends on whether AniDB / Anilist / MAL recognized those as anime. As long as there are entries in these database, it'll be indexed to database, sooner or later.
